### PR TITLE
[Build] Cleanup build yml for handlers

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

Try to cleanup our yml ci

### Issues Resolved ### 

- None

### API Changes ###

 
 None

### Platforms Affected ### 

- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: eng/pipelines/handlers.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - README.md
+    - SECURITY.md
+    - THIRD-PARTY-NOTICES.TXT
+
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - main
```
Comment (PureWeen) [inline eng/pipelines/handlers.yml:44]: should this be main-handler ?

Comment (rmarinho) [reply eng/pipelines/handlers.yml:44]: my idea is this is the main on dotnet/maui

**Comment by rmarinho** [comment]
Unit tests are not running :(

**Review by StephaneDelcroix — APPROVED** [review]

**Pull request merged**